Assets/Scripts/TextDecoder/DirectorActionDecoder.cs

```
@@ -372,6 +398,45 @@ void Wait(string seconds)
         }
     }
 
+    /// <summary>
+    /// Jump-cuts the camera to the target sub position if the bg-scene has sub positions.
+    /// </summary>
+    /// <param name="position">String containing an integer referring to the target sub position, 1 based.</param>
+    void JumpToSubPosition(string position)
+    {
+        if (!HasSceneController())
+            return;
+
+        if (int.TryParse(position, out int pos))
+        {
+            _sceneController.SetToSubPosition(pos);
+        }
+        _onActionDone.Invoke();
+    }
+
+    /// <summary>
+    /// Pans the camera to the target sub position if the bg-scene has a sub positions.
+    /// </summary>
+    /// <param name="positionAndTime">String containing an integer referring to the target sub position, 1 based, and a floating point number referring to the amount of time the pan should take in seconds.</param>
+    void PanToSubPosition(string positionAndTime)
+    {
+        if (!HasSceneController())
+            return;
+
+        string[] parameters = positionAndTime.Split(ACTION_PARAMETER_SEPARATOR);
+
+        if (parameters.Length != 2)
+        {
+            Debug.LogError("Invalid amount of parameters for function PAN_TO_POSITION");
+            return;
+        }
+        if (int.TryParse(parameters[0], out int pos) && float.TryParse(parameters[1], NumberStyles.Any, CultureInfo.InvariantCulture, out float seconds))
```

Nit: This can silently fail if 2 parameters exist but can't be parsed to an integer.  
Suggestion: Invert and split these `if`s into individual conditions and handle failed parses with a call to `Debug.LogError()`.